core/src/mindustry/world/blocks/liquid/Conduit.java

```diff
@@ -119,6 +119,16 @@ public void onProximityUpdate(){
             blending = bits[4];
         }
 
+        @Nullable
+        public ConduitBuild nextConduit(){
+            Tile next = tile.nearby(rotation);
+            if(next != null && next.build instanceof ConduitBuild){
+                return (ConduitBuild) next.build;
+            }else{
+                return null;
+            }
```

Comment:
All this could be shortened by using ternary:

`return next != null && next.build instanceof ConduitBuild ? (ConduitBuild) next.build : null;`